MonoGame.Framework/Content/ContentManager.cs

```diff
@@ -254,8 +255,14 @@ protected virtual Stream OpenStream(string assetName)
 			try
             {
                 string assetPath = Path.Combine(RootDirectory, assetName) + ".xnb";
-                stream = TitleContainer.OpenStream(assetPath);
 
+                // For Win8 metro applications you do not have access to Environment.CurrentDirectory nor does Path.DirectorySeparatorChar exist.
+#if !WINRT
+                if (Path.IsPathRooted(assetPath))
+                    assetPath = FileHelpers.GetRelativePath(Environment.CurrentDirectory + Path.DirectorySeparatorChar, assetPath);
```

Comment:
This fix won't work if the absolute path is on another drive letter.

I suspect we need to just not call `TitleContainer.OpenStream` if the path is absolute... instead calling `File.OpenRead()` directly.

Also this should only be allowed on desktop platforms Linux, MacOS, Windows (not metro) and Android according to https://github.com/mono/MonoGame/issues/3165.